docs/en/flash.md

```diff
@@ -28,7 +28,9 @@ Run the following command to flash an *aggregated* binary as is produced for exa
 - `size` is given in bits. Specify `4m` for 512&nbsp;kByte and `32m` for 4&nbsp;MByte.  If unsure, try a smaller size;
```

Comment:
I think that I would recommend leaving the parameter out. However, you should probably note that the esptool is under active development and that if it doesn't work, the first step should be to upgrade to the current version.